android/quest/src/main/assets/configs/app/profiles/default_profile_config.json

```diff
@@ -2235,6 +2235,7 @@
           "workflow": "LAUNCH_QUESTIONNAIRE",
           "questionnaire": {
             "id": "3276f55c-b25e-455b-ae4e-8846fb8fd039",
+            "type": "EDIT",
```

Comment:
I believe that this should be the fix for the conversation that we were having on Slack?